File: BSharp/Controllers/Dto/SettingsForClient.cs
using System;

namespace BSharp.Controllers.Dto
{
    public class SettingsForClient
    {
        public string ShortCompanyName { get; set; }

        public string ShortCompanyName2 { get; set; }

        public string PrimaryLanguageId { get; set; }

        public string PrimaryLanguageName { get; set; }

        public string PrimaryLanguageSymbol { get; set; }

        public string SecondaryLanguageId { get; set; }

        public string SecondaryLanguageName { get; set; }

        public string SecondaryLanguageSymbol { get; set; }

        public string TernaryLanguageId { get; set; }

        public string TernaryLanguageName { get; set; }

        public string TernaryLanguageSymbol { get; set; }

        public string BrandColor { get; set; }
               
        public DateTimeOffset ProvisionedAt { get; set; }
    }
}

File: Tellma/ClientApp/src/app/data/dto/settings-for-client.ts
import { Calendar, HmsFormat, YmdFormat } from '../entities/base/metadata-types';

// tslint:disable:variable-name
export interface SettingsForClient {
    ShortCompanyName: string;
    ShortCompanyName2: string;
    ShortCompanyName3: string;
    FunctionalCurrencyId: string;
    FunctionalCurrencyDecimals: number; // decimal places of the functional currency
    FunctionalCurrencyName: string;
    FunctionalCurrencyName2: string;
    FunctionalCurrencyName3: string;
    FunctionalCurrencyDescription: string;
    FunctionalCurrencyDescription2: string;
    FunctionalCurrencyDescription3: string;
    ArchiveDate: string;
    TaxIdentificationNumber?: string;
    PrimaryLanguageId: string;
    PrimaryLanguageName: string;
    PrimaryLanguageSymbol: string;
    SecondaryLanguageId: string;
    SecondaryLanguageName: string;
    SecondaryLanguageSymbol: string;
    TernaryLanguageId: string;
    TernaryLanguageName: string;
    TernaryLanguageSymbol: string;
    PrimaryCalendar: Calendar;
    SecondaryCalendar: Calendar;
    DateFormat: YmdFormat;
    TimeFormat: HmsFormat;
    BrandColor: string;
    CreatedAt: string;
    SingleBusinessUnitId?: number;
    SmsEnabled: boolean;
}
